core-bundle/src/Resources/contao/elements/ContentModule.php

```diff
@@ -62,6 +62,12 @@ public function generate()
 		$objModel->cssID = $cssID;
 		$objModel->typePrefix = 'ce_';
 
+		if (System::getContainer()->getParameter('kernel.debug'))
+		{
+			$objStopwatch = System::getContainer()->get('debug.stopwatch');
```

Comment:
Is this service guaranteed to be always there if `kernel.debug` is `true`? Even in custom Symfony applications where Contao is only used as a bundle?

Reply:
It's always there if the `Stopwatch` class exists (https://github.com/symfony/symfony/blob/5.x/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L938) and stopwatch is required by Contao (https://github.com/contao/contao/blob/4.9/composer.json#L107)

Reply:
Curiously I have a Contao Manged Edition `4.9.16` installation where I now get this error:

```
Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException:
The "debug.stopwatch" service or alias has been removed or inlined when the container was compiled. You should either make it public, or stop using the container directly and use dependency injection instead.

  at vendor\symfony\dependency-injection\Container.php:275
  at Symfony\Component\DependencyInjection\Container->make()
     (vendor\symfony\dependency-injection\Container.php:231)
  at Symfony\Component\DependencyInjection\Container->get()
     (vendor\contao\core-bundle\src\Resources\contao\library\Contao\Controller.php:602)
  at Contao\Controller::getContentElement()
     (vendor\contao\core-bundle\src\Resources\contao\dca\tl_content.php:1289)
  at tl_content->addCteType()
     (vendor\contao\core-bundle\src\Resources\contao\drivers\DC_Table.php:4573)
  at Contao\DC_Table->parentView()
     (vendor\contao\core-bundle\src\Resources\contao\drivers\DC_Table.php:365)
  at Contao\DC_Table->showAll()
     (vendor\contao\core-bundle\src\Resources\contao\classes\Backend.php:644)
  at Contao\Backend->getBackendModule()
     (vendor\contao\core-bundle\src\Resources\contao\controllers\BackendMain.php:167)
  at Contao\BackendMain->run()
     (vendor\contao\core-bundle\src\Controller\BackendController.php:48)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor\symfony\http-kernel\HttpKernel.php:158)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw()
     (vendor\symfony\http-kernel\HttpKernel.php:80)
  at Symfony\Component\HttpKernel\HttpKernel->handle()
     (vendor\symfony\http-kernel\Kernel.php:201)
  at Symfony\Component\HttpKernel\Kernel->handle()
     (web\index.php:34)
```